coremltools/converters/mil/frontend/torch/test/test_torch_ops.py

```diff
@@ -4590,6 +4590,30 @@ def forward(self, x):
             frontend=frontend,
         )
 
+    @pytest.mark.parametrize(
+        "compute_unit, backend, frontend",
+        itertools.product(
+            compute_units,
+            backends,
+            frontends,
+        ),
+    )
+    def test_randn(self, compute_unit, backend, frontend):
+        class TestModel(torch.nn.Module):
+            def forward(self, x):
+                noise = torch.randn(x.shape)
```

Comment:
Are we testing both branches (#inputs = 1 or > 1) in this test? If not, it's not clear which case is being tested here.

Reply:
Yes, different `frontend` will have different number of inputs.